libs/island-ui/core/src/lib/RadioButton/RadioButton.tsx

```diff
@@ -129,6 +136,17 @@ export const RadioButton = ({
             <Tooltip text={tooltip} />
           </div>
         )}
+        {tag && large && (
+          <Box
+            display="flex"
+            justifyContent="flexEnd"
+            style={{ width: '-webkit-fill-available' }}
```

Comment:
This is probably not a good idea, because this will only work on Chrome. I think something like `width="full"` would be better.

Reply:
yes, ofcourse! good spot :)